build.py

```diff
@@ -1,25 +1,20 @@
+import os
 import sys
+import subprocess
 from pathlib import Path
 from contextlib import chdir
-import subprocess
-import os
 
-
-def usage() -> None:
-    """Print usage information for the script."""
-
-    print("Usage: python build.py [OPTIONS]", end="\n\n")
+def usage():
```

Comment:
Why remove the typehint on return type?

Reply:
Just my python style, I'm not used to declare return type. You can reject without problems.